### ISSUE TITLE
Domyślne rozwijanie wszystkich grup komentarzy na głównej

### ISSUE THREAD

**Issue by Ozjasz-Goldberg** [wykopx/WykopX#66]
Funkcja potrzebna by móc wygodnie korzystać np. z "ctrl + F" i na szybko przeglądać rozwój dyskusji w całości.

**Comment by wykopx** [comment]
Nie mam pojęcia o co chodzi w tym requescie. 
Co to jest grupa komentarzy na głównej?

Na głównej nie ma żadnych komentarzy tylko lista znalezisk

Napisz więcej info, dokladnie o co chodzi, najlepiej ze zrzutem ekranu.

**Comment by Ozjasz-Goldberg** [comment]
> Nie mam pojęcia o co chodzi w tym requescie. Co to jest grupa komentarzy na głównej?
> Na głównej nie ma żadnych komentarzy tylko lista znalezisk
> Napisz więcej info, dokladnie o co chodzi, najlepiej ze zrzutem ekranu.

Chodziło mi o rozwijanie komentarzy w znaleziskach. Użyłem słowa "główna" by oddzielić tę kwestię od zwiniętych wpisów na mirko. 
![obraz](https://user-images.githubusercontent.com/124906120/221439167-05963495-9322-47a7-8f5c-09a08184e9ba.png)

**Comment by wykopx** [comment]
Czyli chodzi o to, żeby automatycznie był kliknięty przycisk "pokaż komentarze", o ile dobrze rozumiem?

**Comment by Ozjasz-Goldberg** [comment]
> Czyli chodzi o to, żeby automatycznie był kliknięty przycisk "pokaż komentarze", o ile dobrze rozumiem?

Tak. By wszystkie zwinięte komentarze były domyślnie rozwinięte. Wtedy używanie ctrl+f będzie miało jakikolwiek sens.
